src/js/port_handler.js

```diff
@@ -118,14 +119,19 @@ PortHandler.updateCurrentSerialPortsList = async function () {
     const ports = await serial.getDevices();
     const orderedPorts = this.sortPorts(ports);
     this.portAvailable = orderedPorts.length > 0;
-
+    if (this.portAvailable) {
+        console.log(`${this.logHead} Found serial port`, orderedPorts[0]);
+    }
```

Comment:
It's the IF totally necessary? Having a lot of logging with conditions make the code harder. We have several options:
- Log always the value
```suggestion
   console.log(`${this.logHead} Found serial port`, orderedPorts);
```
- Log with a ternary:
```suggestion
   this.portAvailable ? console.log(`${this.logHead} Found serial port`, orderedPorts) : null;
```
- Logic && operator:
```suggestion
   this.portAvailable && console.log(`${this.logHead} Found serial port`, orderedPorts);
```
- Have a utility function to do this that receives the condition and the message:
```suggestion
   logIf(this.portAvailable, `${this.logHead} Found serial port`, orderedPorts);
```

This applies to the rest of the changes for log in this PR. Being console logs, I think we can log them always.

Reply:
Thanks, Removed conditions